pkg/models/sidecar.go

```diff
@@ -2,11 +2,15 @@
 // See the file LICENSE for licensing terms.
 package models
 
+import "github.com/ava-labs/avalanchego/ids"
+
 type Sidecar struct {
-	Name      string
-	VM        VMType
-	Subnet    string
-	TokenName string
-	ChainID   string
-	Version   string
+	Name         string
+	VM           VMType
+	Subnet       string
+	TokenName    string
+	ChainID      string
+	Version      string
+	SubnetID     ids.ID
```

Comment:
we might want to update `subnetcmd/describe.go`

Reply:
Added to `describe` command